.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
```

Comment:
Ah, it maybe that `latexmk` doesn't go the whole way and is only useful for the common case of small changes. In that case stick with `make`, sure.